src/core/AutomationPattern.cpp

```diff
@@ -781,6 +781,16 @@ void AutomationPattern::resolveAllIDs()
 							{
 								a->addObject( dynamic_cast<AutomatableModel *>( o ), false );
 							}
+                            else
```

Comment:
Make sure there are tabs and not spaces in front of the else.

Reply:
This PR seem to be closed.

Reply:
Yeah I closed it because it added like a huge amount of files so to make sure I don't break anything I prefered to close it, and I'll try again the way you explained when I'll have be on my computer